.codeclimate.yml

```diff
@@ -8,11 +8,18 @@ engines:
     checks:
       Drupal Commenting DocComment ShortNotCapital:
         enabled: false
+      Drupal NamingConventions ValidFunctionName NotCamelCaps:
```

Comment:
@sherakama do you want these in here as well, or take them out now that `Controversial/CamelCaseParameterName` is disabled?

Reply:
Leave em. They are in our default config anyhow.

Reply:
**_FINE._**